gamemode/modules/fadmin/fadmin/messaging/sh_shared.lua

```diff
@@ -26,11 +26,11 @@ function FAdmin.TargetsToString(targets)
         return FAdmin.PlayerName(targets)
     end
 
-    local targetCount = #targets
-    if targetCount == 0 then
+    if table.IsEmpty(targets) then
```

Comment:
Dont use IsEmpty here since you use the length below.

Reply:
I was debating not doing this, as if the table is empty is will be faster but trades of performance if they array is filled. As you have complained I will remove